dev-lang/lua/patches/lua-5.3.3.patchset

```diff
@@ -1,8 +1,13 @@
-From b1db2db24aca8b1f8a59949f78cc2cbe0d01bfb8 Mon Sep 17 00:00:00 2001
-From: begasus <begasus@gmail.com>
-Date: Sat, 24 Dec 2016 15:45:36 +0100
-Subject: import patches from 5.2.1
+From b7073ba00e3fcc6b6035fe791f19bf59bbb67137 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sat, 21 Jan 2017 09:11:37 +0100
+Subject: [PATCH] import patches from 5.2.1
```

Comment:
I would have preferred a separate commit in the patchset, rather than modifying the existing commit.
This helps tracking down what changed in the patch, and when.